openscap.yaml

```diff
@@ -81,6 +81,8 @@ subpackages:
         - openscap
         - py3-docker
         - docker-cli
+        # Shbang hardcoded to #!/usr/bin/python3
+        - python3
```

Comment:
fix the shbang.
this is wrong, as it does _not_ depend on "any old python3", it is explicitly installed into one.